generatePlayCards.py

```diff
@@ -61,7 +63,8 @@ def add_text_box(c, info, position, box_size,
         c.setFillColorRGB(r, g, b)
         c.rect(x, y, box_size, box_size, fill=1)
     else:
-        c.rect(x, y, box_size, box_size)
+        if noborder == False: # do not print a border when no color is used for background
```

Comment:
I changed the position because I did a rework and added colored backgrounds. I thought it makes only sense to rmove the rect when no backcol is set, how ould you do it with a laser cutter with backcol? You think it would work good enough?